hasjob/static/js/app.js

```diff
@@ -184,6 +184,192 @@ window.Hasjob.PaySlider.prototype.resetSlider = function(currency) {
 };
 
 $(function() {
+  //Change site button to filter icon
+  $(".hg-site-nav-toggle").find("i").removeClass("fa-bars").addClass("fa-search");
+  $("#hg-sitenav").on("shown.bs.collapse", function() {
+    $(".hg-site-nav-toggle").find("i").removeClass("fa-search").addClass("fa-close");
+  });
+  $("#hg-sitenav").on("hidden.bs.collapse", function() {
+    $(".hg-site-nav-toggle").find("i").removeClass("fa-close").addClass("fa-search");
+  });
+
+  //Add Cancel button for dropdown filter in mobile
+  $("#job-filters-cancel").click(function(event) {
+    event.preventDefault();
+    $("#hg-sitenav").collapse("toggle");
+    $('body').removeClass('nav-open');
+  });
+  
+  var scrollheight = $("#hgnav").height() - $("#hg-sitenav").height();
+  $(window).scroll(function() {
+    if($(window).width() > 767) {
+      if ($(this).scrollTop() > scrollheight) {
+        $(".header-section").slideUp();
+      }
+      else{
+        $(".header-section").slideDown();
+      }
+    }
+  });
+
+  //If Hasjob.PayFilterParameters is empty then assign default values
+  if (typeof window.Hasjob.PayFilterParameters === "undefined") {
```

Comment:
- Move this closer to where it's actually used?
- The comment isn't really needed, since the line below is fairly self-explanatory. What would help is, a reference `index.html`, since that's where the values are set if they are present.